vue.js

```diff
@@ -0,0 +1,46 @@
+/** @type {import("eslint").Linter.Config}  */
+const vue = {
+  extends: [
+    'plugin:vue/vue3-recommended',
+    'eslint:recommended',
+    '@sparing-software/eslint-config',
+    '@vue/eslint-config-typescript/recommended',
```

Comment:
Isn't it conflicting with our eslint typescript configuration?

Reply:
@adrianklimek  switching order causes a lots of weird errors here so i extended the rules manually to take priority. alternatively i could remove the custom vue config and try to configure it from scratch but i'm not sure if that's a better idea.